onset_maproom/calc.py

```diff
@@ -18,13 +34,38 @@ def water_balance(
     sminit,
     time_coord="T",
 ):
-    """Estimates soil moisture from
-    rainfall,
-    evapotranspiration,
-    total available water and
-    intial soil moisture value, knowing that:
-    sm(t) = sm(t-1) + rain(t) - et(t)
-    with roof and floor respectively at taw and 0 at each time step.
+    """ Calculate soil moisture.
```

Comment:
Is there a reason you've added a space after the quotes in each docstring? I don't that's typical.

Reply:
I do not think there was a specific reason, no. I think I just started doing it for some reason..